README.md

```diff
@@ -20,75 +20,79 @@ We plan to iteratively expand the replica’s capabilities, to make the conversa
 This software and the sample data are under very permissive Creative Commons Zero v1.0 Universal license
 (basically, it describes a release into the Public Domain). 
 
-## 1. How to use
+## 1. How to use: Prerequisites
 
 Clone this repo.
 
 Navigate to the AI-replica dir.
 
-Run the console bot:
-`python3 console_bot.py`
+Install `make` tool, so that you can execute useful commands from project's `makefile`.
 
-### Server bot
-Run the server bot (default port is 8000, default address is localhost):
-`python3 server_bot.py`
+## 2. How to use: Console bot
 
-or:
-`python3 server_bot.py --port=8000`
+Make sure that [Prerequisites ](#1-how-to-use-prerequisites) are satisfied.
 
-For help, run:
-`python3 server_bot.py -h`
+Run the console bot:
+`make run_console_bot`
 
-By default, Rasa bot engine is used. Before running the server, make sure that Rasa server is being run at `http://localhost:8002`. Can be changed in `config.yaml`. Also make sure that Rasa actions server is being run at `http://localhost:8004`. 
-To turn off Rasa engine, change the value of bot_engine setting in `config.yaml`.
+## 3. How to use: Server bot
 
-### Web chat
-Build web chat by running `python3 build_web_chat.py`. The web chat will be prepared in the `dist` folder.
+Make sure that [Prerequisites ](#1-how-to-use-prerequisites) are satisfied.
 
-Then open your browser and navigate to the address and port server listens on (by default, the port is 8000).
+By default the server uses Rasa. 
+Install Rasa if not installed. You can do it manually (see [Rasa docs](./documentation/rasa.md)) or run the following command:
+`make install_replica_dependencies`.
 
-### Rasa
-**IMPORTANT.** Install Python 3.8 first as Rasa does not support later versions yet (as of January 2022).
+Build UI chat by runnning:
+`make build_ui`
 
-Go to "rasa" folder, create virtual environment and switch to the created virtual environment: 
-```
-cd rasa
-python3.8 -m venv ./venv
-source ./venv/bin/activate
-```
+Then you can launch all the servers by running:
+`make start_all`
```

Comment:
The command `make start_all`  fails with the error message message `No module named 'psutil'`

Which is strange, as it is listed in requirements.txt

Installing it manually (in the Replica's venv) fixes the problem:
`pip3 install psutil`

Makes sense to add it to the Readme too.

Reply:
Added.